### PR TITLE
Update styles in Trading Page Mobile - Outcome selector and dropdown interaction states

### PR DESCRIPTION
**Pull request by rubyweb3**
https://github.com/AugurProject/augur/issues/5018 
https://github.com/AugurProject/augur/issues/5009

- Update arrow style
- BG shadow when dropdown is active
- Update Order ticket outcome dropdown interaction states

Mobile
Screen Shot 2020-01-14 at 1 56 56 AM
Screen Shot 2020-01-14 at 1 56 23 AM

Desktop
Screen Shot 2020-01-14 at 2 35 03 AM
Screen Shot 2020-01-14 at 2 34 58 AM
Screen Shot 2020-01-14 at 2 34 52 AM

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). 🌠
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/vv2h1p2**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

File: packages/augur-ui/src/modules/common/selection.styles.less [hunk 1]
```diff
@@ -27,30 +27,34 @@
...
-      }
     }
   }
 
   > div:first-of-type {
-    background-color: @color-primary-text;
+    background-color: @color-secondary-action;
+    border: 1px solid @color-secondary-action-outline;
     border-radius: @border-radius-default;
+    box-shadow: 0px @size-4 @size-4 rgba(0, 0, 0, 0.25);
```
Comment (JohnDanz) [inline packages/augur-ui/src/modules/common/selection.styles.less]: we don't use rgba and we also don't want to have a black color like that, can we use something like `fade(@color-dark-grey, 25%);` instead here or even a darker value thats close to black? in general we always want to take from the color pallet we already have in `_colors.less`

**Review by JohnDanz — CHANGES_REQUESTED** [review]
one change request here, otherwise looks good.

File: packages/augur-ui/src/modules/common/selection.styles.less (hunk 1 above)
Comment (rubyweb3) [reply packages/augur-ui/src/modules/common/selection.styles.less]: Sure. I have updated the box-shadow to ```0 @size-8 @size-8 @color-dark-grey;```

```fade(@color-dark-grey, 25%);``` looks not dark enough.

**Review by JohnDanz — APPROVED** [review]

**Pull request merged**